src/main.rs

```diff
@@ -70,6 +70,10 @@ struct Cli {
     #[arg(short, long, value_name = "PATH", env = "AUTOBIB_CONFIG_PATH")]
     config: Option<PathBuf>,
 
+    /// Do not prompt for user action or open the editor.
```

Comment:
Is 'open the editor' redundant here? Perhaps it falls under 'user action'. Maybe `Do not require user action`?